torch/distributed/_sharded_tensor/api.py

```diff
@@ -127,17 +127,22 @@ def _register_remote_shards(sharded_tensor_id: int, rrefs: List[rpc.RRef[Shard]]
 
 class CreateOp(Enum):
     EMPTY = 0
-    ONES = 1
+    FULL = 1
+    ONES = 2
+    RAND = 3
+    ZEROS = 4
 
 
 @dataclass
 class TensorInitParams(object):
     """ Container for list of common params to create new local tensor. """
 
-    __slots__ = ['create_op', 'dtype', 'layout', 'requires_grad', 'pin_memory',
-                 'memory_format']
+    __slots__ = ['create_op', 'fill_value', 'dtype', 'layout', 'requires_grad',
+                 'pin_memory', 'memory_format']
 
     create_op: CreateOp
+    # needed when create_op is FULL
+    fill_value: Number
```

Comment:
Since this is optional, can we specify a default value: `fill_value: Number = None`, this way users don't need to pass this in unless they are using `CreateOp::FULL`.

Reply:
last time I got some flake8 complaint, but yes it made sense to set default as None